libpyvinyl/DiffractionData/testDiffractionData.py

```diff
@@ -0,0 +1,12 @@
+
+
+from libpyvinyl.DiffractionData import DiffractionData
+import numpy as np
+
+arr = np.arange(9).reshape([3, 3])
+distance = 10
+diffraction_data = DiffractionData(arr, 10)
+diffraction_data.write('test.h5', 'simple')
+data_from_file = DiffractionData.read('test.h5', 'simple')
```

Comment:
The initial intention of using a string instead of the exact class variable is to avoid 'import SimpleFormat' to simplify the usage. @CFGrote Could there be a better way?